docs(*): add more tips while using multiple domains (#8059)
Kong/docs.konghq.com: app/_src/gateway/kong-manager/enable.md

  admin_gui_path = /manager
  ```
> Make sure that each domain has proper DNS records and that the Kong instance is accessible from all specified domains.

{:.important}
> **Important**: If your setup involves multiple domains or subdomains, it’s generally recommended to remove the `cookie_domain` that setting in the `admin_gui_session_conf` or `admin_gui_auth_conf`.

> > When `cookie_domain` is not specified, cookies are set for the origin domain by default. This allows the browser to manage cookies correctly for each domain independently, avoiding conflicts or scope issues. For example:

Reviewer: I realized that the cookies will be set for `admin_gui_api_url` if it's specified but `cookie_domain` is not, and `admin_gui_api_url` is not necessary to be affiliated with `admin_gui_url`. To make it more accurate:

```suggestion
> When `cookie_domain` is not specified, cookies are set for the domain initiated the request, if `admin_gui_api_url` is not specified. This allows the browser to manage cookies correctly for each domain independently, avoiding conflicts or scope issues. For example:
```

>

> > Example A: A request to `gui.konghq.com`/`other-gui.example.com` will only receive cookies for `gui.konghq.com`/`other-gui.example.com` instead of a broader `konghq.com`/`example.com` domain if `cookie_domain` is omitted

Reviewer: I guess we intended to compare the cookies for subdomains (`gui.konghq.com`/`other-gui.konghq.com`) with the cookies for the root-level domain (`konghq.com`) here?

```suggestion
> Example A: Requests to `gui.konghq.com` and `other-gui.konghq.com` will produce cookies for `gui.konghq.com` and `other-gui.konghq.com` respectively, instead of ones for the root-level `konghq.com` domain when `cookie_domain` is not specified.
```

> > Example A: Requests to `gui.konghq.com` and `other-gui.example.com` will produce cookies for `gui.konghq.com` and `other-gui.example.com` respectively, instead ones of the root-level `konghq.com` domain when `cookie_domain` is not specified.
>   ```
>   admin_gui_url = http://gui.konghq.com, http://other-gui.example.com
>   admin_gui_session_conf = {"secret":"Y29vbGJlYW5z","storage":"kong","cookie_secure":false} # omitted `cookie_domain`
>   ```

Reviewer: In Example B, we took gui.**konghq.com** and other-gui.**konghq.com** as examples, where they share a root-level domain–`konghq.com`. 

However, we used gui.**konghq.com** but other-gui.**example.com** here in Example A, where two domains **do not** share the same root-level domain.

If we would like to demonstrate that missing `cookie_domain` will make the session cookie across multiple domains not work even if they share the same root-level domain–like the gui.**konghq.com** and other-gui.**konghq.com** case, shall we use the same setup and only let the `cookie_domain` be the variable?

```suggestion
> Example A: Requests to `gui.konghq.com` and `other-gui.konghq.com` will produce cookies for `gui.konghq.com` and `other-gui.konghq.com` respectively, instead of ones for the root-level `konghq.com` domain when `cookie_domain` is not specified.
  ```
  admin_gui_url = http://gui.konghq.com, http://other-gui.konghq.com
  admin_gui_session_conf = {"secret":"Y29vbGJlYW5z","storage":"kong","cookie_secure":false} # omitted `cookie_domain`
  ```
```

> > Example B: A request to `gui.konghq.com`/`other-gui.konghq.com` will receive cookies for `konghq.com`. This allows the cookie to be shared across all subdomains, but be cautious since it increases the cookie’s scope, which may lead to unintended side effects or security risks.

Reviewer: ```suggestion
> Example B: Both requests to `gui.konghq.com` and `other-gui.konghq.com` will receive cookies for `konghq.com`, which makes the cookie shared across all subdomains besides `konghq.com` itself. Please be cautious since it increases the cookie's scope, which may lead to unintended side effects or security risks.
```

  ```
  admin_gui_url = http://gui.konghq.com, http://other-gui.konghq.com

>   admin_gui_session_conf = {"secret":"Y29vbGJlYW5z","storage":"kong","cookie_secure":false,"cookie_domain" : "konghq.com"}

Reviewer: nit

```suggestion
  admin_gui_session_conf = {"secret":"Y29vbGJlYW5z","storage":"kong","cookie_secure":false,"cookie_domain":"konghq.com"}
```
